src/mono_camera_node.cpp

```diff
@@ -40,7 +42,10 @@ MonoCameraNode::MonoCameraNode() : Node("camera"), api_(this->get_logger()), cam
   camera_info_pub_ = image_transport::create_camera_publisher(this, "~/image", rmw_qos_profile_system_default);
 
   // Set the frame callback
-  cam_.setCallback(std::bind(&avt_vimba_camera::MonoCameraNode::frameCallback, this, std::placeholders::_1));
+  cam_.setCallback(std::bind(&avt_vimba_camera::MonoCameraNode::frameCallback, this, _1));
+
+  start_srv_ = create_service<std_srvs::srv::Trigger>("start_stream", std::bind(&MonoCameraNode::startSrvCallback, this, _1, _2, _3));
```

Comment:
Should the service names be "~/start_stream" and "~/stop_stream" so they are associated with the correct namespace?

Reply:
You're right, I forgot about this. Changes applied ;)